Formula/srtp.rb

```diff
@@ -22,9 +22,13 @@ class Srtp < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "openssl@1.1"
 
   def install
-    system "./configure", "--disable-debug", "--prefix=#{prefix}"
+    ENV.prepend "LDFLAGS", "-L#{Formula["openssl@1.1"].opt_lib}"
+    ENV.prepend "CPPFLAGS", "-I#{Formula["openssl@1.1"].opt_include}"
```

Comment:
Are these really needed?

Reply:
I'll check. I grabbed this from the `postgresql` recipe, but just noticed with a different recipe I'm working on that `depends_on` also linked things correctly.

Reply:
You were right that these were unnecessary. I took them out in the commit I just pushed.